tests/unit/command/test_cache.py

```diff
@@ -0,0 +1,40 @@
+import os
+import pytest
+from mock import MagicMock
+
+from dvc.config import Config
+
+
+@pytest.mark.skipif(os.name == "nt", reason="Linux only")
+def test_path_replacement():
```

Comment:
Let's move these tests to `tests/unit/test_config.py` please.

Reply:
Sure!